core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -933,6 +934,9 @@ object KafkaConfig {
   val CompressionTypeDoc = "Specify the final compression type for a given topic. This configuration accepts the standard compression codecs " +
   "('gzip', 'snappy', 'lz4', 'zstd'). It additionally accepts 'uncompressed' which is equivalent to no compression; and " +
   "'producer' which means retain the original compression codec set by the producer."
+  val CompressionLevelDoc = "The compression level for all data generated by the producer. The default level and valid value is up to " +
+    "compression.type. (<code>none</code>, <code>snappy</code>: not available. <code>gzip</code>: 1~9. <code>lz4</code>: 1~17. " +
+    "<code>zstd</code>: -131072~22."
```

Comment:
Since the docs use `[min,...,max]` syntax for things validated by `ConfigDef.Range` I think we should use a similar syntax here for consistency (even though it's not validated by `Range`).